tests/benchmarks/multicore/idle_ppr/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&cpuppr_vpr {
```

Comment:
Check with PPR snippet

Reply:
Can't use PPR snippet due to resources collision. App reserves uart135 for PPR
https://github.com/nrfconnect/sdk-zephyr/blob/main/snippets/nordic-ppr/boards/nrf54h20dk_nrf54h20_cpuapp.overlay#L10
while Radio core uses that uart instance:
https://github.com/nrfconnect/sdk-zephyr/blob/main/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad.dts#L97